api/app/User.php

```diff
@@ -44,4 +44,16 @@ public function isAdmin()
     {
         return $this->is_admin;
     }
+
+    public function passwordResets()
+    {
+        return $this->hasMany('App\PasswordReset');
+    }
+
+    public function passwordresets()
```

Comment:
2 functions, one with captial letter and one without. Quit confusion if someone needs to use it.